pkg/grpc/grpc.go

```diff
@@ -87,6 +88,10 @@ func NewGRPCServersFromConfigurationAndServe(configurations []*configuration.GRP
 		grpc_prometheus.EnableHandlingTimeHistogram(grpc_prometheus.WithHistogramBuckets(prometheus.DefBuckets))
 		grpc_prometheus.Register(s)
 
+		if len(configuration.ListenAddresses)+len(configuration.ListenPaths) == 0 {
```

Comment:
Out of interest, did you run `gofmt`? I would have expected it to add spaces around the + operator in this case.

Reply:
This is no longer relevant as I changed this conditional, but I did check and gofmt appears to prefer the spaces removed.